src/services/image.service.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { recipes } from "src/test/mocks/recipe.mocks";
```

Comment:
Stick to relative imports (ones with `../test/`)

Reply:
But the way, why are you using mocks in service code?

Reply:
Idk, it must have been automatic import.